docs/setup/jetson.md

```diff
@@ -0,0 +1,162 @@
+<!--
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+# Connect Create® 3 to NVIDIA® Jetson and set up ROS 2 Galactic
+
+## Before you start
+!!! attention
+    **These directions are written for someone with experience with embedded Linux and basic embedded computers.**
+It is highly recommended to read through the getting started document for your Jetson platform before beginning:
+
+* [Getting Started With Jetson Xavier NX Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit)
+* [Getting Started with Jetson Nano Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit)
+* [Getting Started with Jetson Nano 2GB Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-2gb-devkit)
+
+
+## Step-by-step
+
+### Getting Started Guide portion
+
+1. Download the SD card image for your Jetson platform and write onto a microSD card.
+
+2. Complete the getting started guide (listed above) until you complete the initial setup.
+
+### Alter "USB Device Mode" to apply static IP 
+
+1. To alter the Jetson's "USB Device Mode" feature (specifically, disabling DHCP server and self-assign a static IP address);
+
+    1. First, create a flag file to store IP address for Jetson to be used with Create3.
```

Comment:
Create3 -> Create® 3
This applies to all occurrences.

Reply:
Fixed in commit 363a5df